Sources/ExyteChat/Views/MessageView/MessageView.swift

```diff
@@ -226,11 +226,20 @@ struct MessageView: View {
     var avatarView: some View {
         Group {
             if showAvatar {
-                AvatarView(url: message.user.avatarURL, avatarSize: avatarSize)
-                    .contentShape(Circle())
-                    .onTapGesture {
-                        tapAvatarClosure?(message.user, message.id)
-                    }
+                if let url = message.user.avatarURL {
```

Comment:
This is the core of the change if an imageUrl is not set then simply displays the first letter of the users name